webcompat/static/js/lib/issue-list.js

```diff
@@ -84,12 +84,14 @@ issueList.DropdownView = Backbone.View.extend({
 
 issueList.FilterView = Backbone.View.extend({
   el: $('.js-issuelist-filter'),
+  mainView: null,
   events: {
     'click .js-filter-button': 'toggleFilter'
   },
   _isLoggedIn: $('body').data('username'),
   _userName: $('body').data('username'),
-  initialize: function() {
+  initialize: function(theOptions) {
```

Comment:
Your code already used options as a local variable name in this method so I wasn't allowed to :-p
Can change that of course..